### PR TITLE
Clownbugs

### PR DESCRIPTION
**Pull request by Super3222**
### Intent of your Pull Request

Ports KMC's clownbugs from the old repo.

#### Changelog

:cl:
rscadd: Clownbugs have returned.
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
make the grenade spawn 5

File: code/game/objects/items/weapons/grenades/spawnergrenade.dm [hunk 1]
```diff
@@ -43,3 +43,8 @@
 /obj/item/weapon/grenade/spawnergrenade/syndiesoap
 	name = "Mister Scrubby"
 	spawner_type = /obj/item/weapon/soap/syndie
+
+/obj/item/weapon/grenade/spawnergrenade/clownbugs
+	name = "clown bug special"
+	spawner_type = /mob/living/simple_animal/cockroach/clownbug
+	deliveryamt = 20
```
Comment (Cruix) [inline code/game/objects/items/weapons/grenades/spawnergrenade.dm:50]: no

File: code/modules/mob/living/simple_animal/friendly/cockroach.dm [hunk 2]
```diff
@@ -54,3 +54,17 @@
...
+/mob/living/simple_animal/cockroach/clownbug
+	name = "clown bug"
+	desc = "Absolutely disgusting... almost as horrid as that one green clown."
+	icon_state = "clownbug"
+	icon_dead = "clownbug"
+
+/mob/living/simple_animal/cockroach/clownbug/death(gibbed)
+	var/turf/T = get_turf(src)
+	if(T)
+		new /mob/living/simple_animal/cockroach/clownbug(T) //OH GOD NO, WHY.
```
Comment (Cruix) [inline code/modules/mob/living/simple_animal/friendly/cockroach.dm:67]: doubleplus no

Comment (ShadowDeath6) [reply code/modules/mob/living/simple_animal/friendly/cockroach.dm:67]: that's the point of it

Comment (Cruix) [reply code/modules/mob/living/simple_animal/friendly/cockroach.dm:67]: if one walks into space it will create an infinite loop spawning bikehorns

Comment (ShadowDeath6) [reply code/modules/mob/living/simple_animal/friendly/cockroach.dm:67]: intended

Comment (Super3222) [reply code/modules/mob/living/simple_animal/friendly/cockroach.dm:67]: one, that sounds fucking hilarious
and two, i can always make them space immune

Comment (ShadowDeath6) [reply code/modules/mob/living/simple_animal/friendly/cockroach.dm:67]: make them immune to everything that isn't a mob attacking them or getting stomped on

**Comment by Super3222** [comment]
Good luck to all of the new coders looking at this as a tutorial. o7 I'm sure you'll all make amazing features.

**Pull request merged**